templates/component/helm_template/Makefile.tmpl

```diff
@@ -25,3 +25,12 @@ set-eks-context:
 
 lint:
 	@helm lint -f variables.yaml .
+
+fmt:
```

Comment:
These are not well defined yet, so leaving a message and moving on. That way we don't break the top level `make fmt` etc